Assets/BillboardLookAt.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BillboardLookAt : MonoBehaviour
```

Comment:
This script could be good to keep but im not sure if using it on the visualisers are a good idea. Also the indentation, brace style and the file location is wonky. You should probably take a look at these points.

Reply:
can you please explain why it´s not a good idea?

Reply:
Currently progressbars are not billboard. The main issues are  that this results in the bubble sometimes covering the progressbar if the camera is above it. It also moves the speech bubble "indicatior" of who is talking away from the viking.